lib/logger.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+let logger = null;
+
+const set = l => logger = l;
+const get = () => logger;
+
+const isDebug = () => logger && logger.debug;
```

Comment:
That was the actual reason I named them as such, I was mislead by [issue](https://github.com/winstonjs/winston/issues/215) in `winston` that actually suggested to check level this way.
I haven't found a good way to get debug level out of winston beside `.level` which gives a string, that's disappointing. Lucky enough bunyan returns a number which is a lot better. I guess we will allow user to provide a callback to check current log level and I'll write proxies for winston and bunyan to simplify their usage.